templates/kubernetes/managed.html

```diff
@@ -0,0 +1,166 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+{% block title %}Managed Kubernetes | Kubernetes{% endblock %}
+{% block meta_description %}With managed kubernetes, Canonical will build, support and manage your cluster for only $4 per node per day.{% endblock meta_description %}
+{% block extra_body_class %}kubernetes{% endblock %}
+
+
+{% block content %}
+
+  <section class="p-strip is-deep is-bordered">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>Managed Kubernetes</h1>
+        <p>Focus on your business while Canonical takes care of building and running your Kubernetes infrastructure.
+        Canonical will set-up and operate your Kubernetes cluster for you, with in-house experts on-hand at all times to stand-up and scale your cluster. And, when you&rsquo;re ready, we will hand over operational control.</p>
+        <ul>
+          <li>Fully managed Kubernetes on your bare metal, VMWare, OpenStack or any public cloud</li>
+          <li>Built in two weeks using a proven reference architecture with our <a href="/kubernetes">Kubernetes consulting packages</a></li>
```

Comment:
and the link needs a class="p-link--inverted"

Reply:
```
<section class="p-strip--accent is-deep">
    <div class="row u-equal-height u-vertically-center">
      <div class="col-7">
        <h1>Managed Kubernetes</h1>
        <p>Focus on your business while Canonical takes care of building and running your Kubernetes infrastructure.
        Canonical will set-up and operate your Kubernetes cluster for you, with in-house experts on-hand at all times to stand-up and scale your cluster. And, when you’re ready, we will hand over operational control.</p>
        <ul>
          <li>Fully managed Kubernetes on your bare metal, VMWare, OpenStack or any public cloud</li>
          <li>Built in two weeks using a proven reference architecture with our <a class="p-link--inverted" href="/kubernetes">Kubernetes consulting packages</a></li>
          <li>Customise the architecture in iterations based on workloads</li>
          <li>Enterprise class SLA for the infrastructure, upgrades included</li>
          <li>Monitoring and log management integration</li>
          <li>When you’re ready, we will hand over the keys</li>
        </ul>
        <p><a href="/containers/contact-us?product=containers-managed-kubernetes" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Schedule a Kubernetes demo', 'eventLabel' : 'Schedule a Kubernetes demo - Top CTA' : undefined });" class="p-button--positive">Schedule a Kubernetes demo</a></p>
      </div>
      <div class="col-5 u-align--center u-hidden--small">
        <img src="https://assets.ubuntu.com/v1/35b71a5e-K8s-cloud.svg" alt="Kubernetes Managed Service Provider">
      </div>
    </div>
  </section>
```

![image](https://user-images.githubusercontent.com/441217/32551157-7dc74d7e-c487-11e7-8a16-ef981788def7.png)